src/Cthulhu.jl

```diff
@@ -258,7 +266,7 @@ function _descend(term::AbstractTerminal, interp::CthulhuInterpreter, mi::Method
             else
                 (codeinf, rt, infos, slottypes) = lookup(interp, mi, optimize)
             end
-            ci = preprocess_ci!(codeinf, mi, optimize, CONFIG)
+            codeinf = preprocess_ci!(codeinf, mi, optimize, CONFIG)
```

Comment:
Maybe, but see #196. You might test `@descend tryparse(VersionNumber, "v2.1.3")` on this branch.

Reply:
Thanks, I think I have a fix for #196

Reply:
I will address this issue in a separate PR